baseplate/experiments/providers/__init__.py

```diff
@@ -0,0 +1,137 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+
+from .forced_variant import ForcedVariantExperiment
+from .legacy import LegacyExperiment
+from ...features.feature import feature_flag_from_config
+
+logger = logging.getLogger(__name__)
+
+
+class ExperimentManager(object):
```

Comment:
I might just call this "Experiment".  I'm also considering rolling targeting back into experiments and treating feature flags as a type of experiment, in which case this might go away.